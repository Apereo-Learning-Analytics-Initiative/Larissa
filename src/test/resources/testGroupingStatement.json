{
    "actor":{
        "mbox":"mailto:Keith.Metellus@exampleonly.com",
        "name":"Keith Metellus"
    },
    "verb":{
        "id":"http://adlnet.gov/expapi/verbs/asked",
        "display":{
            "en-US":"asked"
        }
    },
    "object":{
        "id":"act:adlnet.gov/course",
        "definition":{
            "type":"type:media",
            "name":{
                "en-US":"course"
            },
            "description":{
                "en-US":"course"
            }
        }
    },
    "context":{
        "contextActivities":{
            "grouping":{
                "id":"act:adlnet.gov/cont2"
            }
        },
        "registration":"ae2128e2-2109-45f3-af23-b27f18a64c1c"
    }
}