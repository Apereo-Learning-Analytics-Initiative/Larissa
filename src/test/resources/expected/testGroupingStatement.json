{
  "actor" : {
    "objectType" : "Agent",
    "name" : "Keith Metellus",
    "mbox" : "mailto:Keith.Metellus@exampleonly.com"
  },
  "verb" : {
    "id" : "http://adlnet.gov/expapi/verbs/asked",
    "display" : {
      "en-US" : "asked"
    }
  },
  "object" : {
    "objectType" : "Activity",
    "id" : "act:adlnet.gov/course",
    "definition" : {
      "name" : {
        "en-US" : "course"
      },
      "description" : {
        "en-US" : "course"
      },
      "type" : "type:media"
    }
  },
  "context" : {
    "registration" : "ae2128e2-2109-45f3-af23-b27f18a64c1c",
    "contextActivities" : {
      "grouping" : [ {
        "objectType" : "Activity",
        "id" : "act:adlnet.gov/cont2"
      } ]
    }
  }
}